Alicerce/Sources/Protocols/ReusableView.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  ReusableView.swift
+//  Alicerce
+//
+//  Created by Luís Afonso on 16/12/2016.
+//  Copyright © 2016 Mindera. All rights reserved.
+//
+
+import UIKit
+
+public protocol ReusableView: View {
+    static var reuseIdentifier: String { get }
+}
+
+public extension ReusableView where Self: UIView {
+    public static var reuseIdentifier: String { return "\(self)" }
+}
+
+public extension ReusableView where Self: UITableViewCell {
+    public init() {
```

Comment:
The use case of this method, is to do cell calculations, mostly. It was useful for me in the past, don't know if it will be used yet, but perhaps we can live it for now and remove it later